### ISSUE TITLE
Installing

### ISSUE THREAD

**Issue by captainnima** [lorint/AdventureWorks-for-Postgres#12]
I am on a Mac. I have everything installed. From Terminal, when I run the psql commands, it asks for my password which I type in but it fails. The problem seems to be the it defaults to my username instead of the Postgres account which is in the database. Any suggestions how I can do this?

**Comment by loganphillips792** [comment]
Can you provide the command you are running?

**Comment by captainnima** [comment]
I run the psql -d command on my Mac. I am logged in as my super user account and when I run the command, it asks for my password. I enter it but it says “Sorry”. I connected to Postgres from the command line but when I do, I am not sure how to run the install.sql. I tried switching to root and running the command but it still asks for the root password and then it fails. I put in the Postgres password also but it fails.
